react/src/context/__tests__/PostHogContext.test.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Thanks, I hadn't realised that React [doesn't actually have a default export](https://github.com/facebook/react/pull/18102). FYI I've discovered that another possible way to deal with this is to set `allowSyntheticDefaultImports: true` in `tsconfig.json`.

Reply:
Yeah, and vaguely remember that * as React is the correct way to import it if you would follow the spec a former colleague once pointed to a stackoverflow page for that.